code/game/machinery/doors/airlock.dm

```diff
@@ -1394,6 +1394,10 @@ About the new airlock wires panel:
 		if(user)
 			to_chat(user, "<span class='notice'>You remove the airlock electronics.</span>")
 		var/obj/item/airlock_electronics/ae
+		if(emagged)
+			electronics = new /obj/item/airlock_electronics/destroyed()
+			operating = FALSE
+
```

Comment:
This might be better placed in the `emag_act` proc, like how you did for windoors. 

![image](https://user-images.githubusercontent.com/42044220/111017550-f040eb80-8379-11eb-9fc1-da77d12774ff.png)

Reply:
This way doesn't make a new object unless you take it apart, which means less load on the server.

Reply:
The amount of load this will make is negligible, like less than 0.1% of the tick usage. Put it in `emag_act()`